metricproxy/exec_proxy.go

```diff
@@ -275,7 +270,7 @@ func newExecCachingProxy(config *config.ExecCachingExporterConfig) *execCachingP
 		resultReadyCh: rdyCh,
 		lastResultMtx: &sync.RWMutex{},
 
-		log: log.Base(),
+		log: zap.L(),
```

Comment:
what the point for saving logger instance if we do not modify it? maybe better `log: zap.L().With(zap.String("component", "proxy"))`

Reply:
this saved instance will be modified later. I did not modify the algorithm to make my change-set minimal